airflow/contrib/operators/mysql_to_gcs.py

```diff
@@ -130,6 +130,12 @@ def _write_local_data_files(self, cursor):
             names in GCS, and values are file handles to local files that
             contain the data for the GCS objects.
         """
+        class BinaryTypeEncoder(json.JSONEncoder):
+            def default(self, obj):
+                if isinstance(obj, binary_type):
+                    return str(obj, 'utf-8')
```

Comment:
`str(object=b'', encoding='utf-8', errors='strict')` is not compatible with py2; so this change would break py2 (you can use `decode()` instead)